tests/utils/test_utils.go

```diff
@@ -146,3 +128,21 @@ func CreateTestWatch(t *testing.T, policyName string, watchName, severity xrayUt
 		assert.NoError(t, xrayManager.DeletePolicy(policyParams.Name))
 	}
 }
+
+func CreateTestProjectEnvInTempDir(t *testing.T, projectPath string) (string, func()) {
```

Comment:
I'd change the name to CreateTestProjectInTempDir
Using 'Env' is usually related to env var and it might be confusing